garden/public/mockServiceWorker.js

```diff
@@ -0,0 +1,292 @@
+/* eslint-disable */
```

Comment:
This file is just something we commit along with `msw` that's only included in dev builds. Only review if you're curious to learn about msw's innards